README.md

```diff
@@ -64,6 +64,39 @@ in the bundle (perhaps because of out-of-date dependencies).
 
 * `--noroot`: By default, source-map-explorer finds common prefixes between all source files and eliminates them, since they add complexity to the visualization with no real benefit. But if you want to disable this behavior, set the `--noroot` flag.
 
+## API
+### `explore(filePathOrContent[, sourceMapPathOrContent][, options])`
```

Comment:
Might want to explicitly say where `explore` came from.

```js
const explore = require('source-map-explorer');
```

Reply:
I feel like this is not mart of the documentation for the function and that like is already long. There is example below that show precisely hot to call it.